install/install_openadapt.ps1

```diff
@@ -31,26 +27,35 @@ $VCRedistRegPath = "HKLM:\SOFTWARE\Microsoft\VisualStudio\14.0\VC\Runtimes\X64"
 function RunAndCheck {
     Param
     (
-        [Parameter(Mandatory = $true)] [string] $Command,
-        [Parameter(Mandatory = $true)] [string] $Desc
+        [Parameter(Mandatory = $true)]
+        [string] $Command,
+
+        [Parameter(Mandatory = $true)]
+        [string] $Desc,
+
+        [Parameter(Mandatory = $false)]
+        [switch] $SkipCleanup = $false
```

Comment:
I think it would be more intuitive for this variable to be renamed, such as to "CleanupAfter"

Reply:
I have used `SkipCleanup` as In some cases I do not want to remove the OpenAdapt folder. Hence, the given name.

`CleanupAfter` is also a nice choice. In my opinion, both imply the same meaning when a developer sees the code.